ui/src/message_popup/playtest/danson_experience_page.jsx

```diff
@@ -69,6 +69,32 @@ export default React.createClass({
     });
   },
 
+  onGetNextQuestion(questions:[QuestionT], responses:[ResponseT]) {
```

Comment:
This looks great, but could use a one line comment at the top of the method about why there's this logic here.